packages/auth/src/components/AuthenticationContainer.tsx

```diff
@@ -68,6 +70,7 @@ const logoutAndRefreshPage = (
       authSessionIsCurrentlyActive = Date.now() < decodedAccessToken.exp * 1000; // JWT expiry date in seconds, not milliseconds
   }
 
+  postLogoutMessage?.();
```

Comment:
Could you add a comment to indidate what this actually does?

Reply:
Added in [document post logout message in logout function](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9504/commits/e6a1af50871bbc6fbdd839e674f464e11bba36fd)